### DISCUSSION TITLE
Title on Highlight block

### DISCUSSION THREAD

**Discussion by eaguad1337** [colinwilson/lotusdocs#126] [Q&A]
Hello, first of all, great work with Lotus Docs! the template is beautiful!

I wonder if it's possible to set a title on a highlight block like this:

![image](https://github.com/colinwilson/lotusdocs/assets/19355144/580b64b0-59a1-41de-8618-4f18ebaf3daf)

I hope there's a way to do it!

EDIT: I used a single tab to make this work.

**Comment by colinwilson** [comment]
Yep. I've thought about implementing this.

It's possible, but turned out to be a little more complicated than I expected. I've used a single tab as a solution too. I may re-visit this when I have the time.
